libsolidity/analysis/OverrideChecker.cpp

```diff
@@ -767,6 +767,39 @@ void OverrideChecker::checkOverrideList(OverrideProxy _item, OverrideProxyBySign
 			}
 		}
 	}
+	else if (specifiedContracts.size() > 1)
+	{
+		vector<int64_t> overrideSpecifierContractIds{};
```

Comment:
```suggestion
		vector<int64_t> overrideSpecifierContractIds;
```
Is this recommended by the unified initialization "guide"?